<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instanceRollingWidget</class>
 <widget class="QWidget" name="instanceRollingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>332</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>332</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>332</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabsTW">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="statusTab">
    <attribute name="title">
     <string>STATUS</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>631</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Instance</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>CONTROL</string>
      </property>
      <widget class="QPushButton" name="startBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="restartBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>RESTART</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>STATUS</string>
      </property>
      <widget class="QRadioButton" name="runningRatio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>IS RUNNING</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="logsEditStatus">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>431</width>
        <height>261</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="debugTab">
    <attribute name="title">
     <string>DEBUG</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>DEBUG</string>
     </property>
     <widget class="QPushButton" name="runCodeBtn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>260</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RUN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadScriptBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD SCRIPT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="codeLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>501</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="logsEditDebug">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>601</width>
        <height>201</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
